### DISCUSSION TITLE
Sudden Increase in Inference Time with YOLOv8-pose

### DISCUSSION THREAD

**Discussion by Salmanshu4** [ultralytics/ultralytics#14981] [Q&A]
I am running YOLOv8-pose for key point detection to identify darts on a dartboard. I’m experiencing some issues with inference time. The model suddenly jumps from 0.22 seconds to 0.83 seconds for the same image, even though it is already warmed up. Has anyone else encountered this problem or have any suggestions on how to address it?

**Comment by pderrenger** [comment]
@Salmanshu4 please ensure you are using the latest version of the Ultralytics package, as updates often include performance improvements and bug fixes. If the issue persists, consider profiling your system to identify any resource bottlenecks. Additionally, you might want to check if any background processes are affecting the inference time. If you continue to experience issues, please provide more details about your setup and environment for further assistance.

**Comment by Salmanshu4** [comment]
@pderrenger I have updated the Ultralytics package, but the issue persists. I am using the following hardware:
01:00.0 3D controller [0302]: NVIDIA Corporation GP102GL [Tesla P40] [10de:1b38] (rev a1)
also I warm-up the model using model.predict with dummy images since the first run takes more time.  Is there any other information you need?”
